### ISSUE TITLE
[rrd4j] add support for DateTimeType

### ISSUE THREAD

**Issue by robnielsen** [openhab/openhab-addons#9292]
rrd4j should be able to support DateTypeType since internally it is just number that represents an amount of time since epoch.

**Comment by kaikreuzer** [comment]
Assuming we map it to the default timezone (i.e. potential time zone information will be lost), this should be feasible.

**Comment by robnielsen** [comment]
@kaikreuzer, I think this is acceptable. Internally, I'd think the time would be stored in GMT time and then mapped back to the default time zone.

**Comment by paulianttila** [comment]
rrd4j

**Comment by robnielsen** [comment]
> How would rrd4j handle average calculations for those timestamps?

Just like any other average, it is just a number.

**Comment by paulianttila** [comment]
@robnielsen, how useful you think that average timestamps would be. E.g. rrd4j default configuration would store exact time stamps from last 8 hours and then start to compress data by average function.

**Comment by robnielsen** [comment]
@paulianttila, isn't this the same issue other data types have as well?

**Comment by paulianttila** [comment]
Yes, but normally you store data in rrd4j which can be compressed over the time like temperatures or humidity levels, etc.

There are other more useful embedded databases already available for this kind of data like sqlite (in jdbc persistence binding), which store values as they are, but if you think that's useful, who am I to judge.

**Comment by lsiepel** [comment]
From the conversation I conclude there does not seem to be a use case for the support of the datetimetype in rrd4j. 
As this issue is 5 years old without a follow-up I think it might be better to close this. 

When one comes with a use case we can re-open and maybe implement such feature.
